manifests/z/ZAP/ZAP/2.13.0/ZAP.ZAP.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+PackageIdentifier: ZAP.ZAP
+PackageVersion: 2.13.0
+PackageLocale: en-US
+Publisher: ZAP
+PublisherUrl: https://github.com/zaproxy/zaproxy
+PublisherSupportUrl: https://github.com/zaproxy/zaproxy/issues
+# PrivacyUrl:
+Author: ZAP
+PackageName: ZAP
```

```suggestion
PackageName: OWASP Zed Attack Proxy
```
This should match Control Panel as closely as possible

---

Fine for it to be "Zed Attack Proxy", but would definitely prefer it not to include "OWASP"

---

This metadata is used to match previous and future versions to this Package identifier.

---

Well, don't you want people to update from this version to the next? if so the PackageName should be updated, or.. you could cheese it by adding the ProductCode,  but then please update the Publisher and PackageName to what it actually will display in the control panel

---

Yes, we want people to be able to update from this version to the next, which is why we dont want to include "OWASP" in the Publisher or Package Name 😁 
If thats a requirement then we'll put this PR on hold until we have a new version...

---

So we can have "OWASP" in the Publisher and Package name for this release but then change it for the next release (which will match the Control Panel) without that causing any problems?

---

Yes, that should work, assuming the control panel entries would be changed in the next release to also be without `OWASP`